lib/menu.ts

```diff
@@ -205,12 +245,19 @@ export const cssMenuItem = styled('li', `
   justify-content: space-between;
   outline: none;
 
-  &-sel {
+  &.menuItemSelected {
```

Comment:
In the spirit of avoiding global css classes, let's use `&-sel` and expose below `const menuItemSelectedClass = cssMenuItem.className + '-sel';`, which can then be used elsewhere in this file. (I don't think it needs to be exported.)